webpack.base.js

```diff
@@ -16,6 +16,18 @@ module.exports = ({ entry, mode, filename, datacenter }) => ({
   devtool: mode === 'development' ? 'inline-source-map' : 'false',
```

Comment:
It is not the first time that we [break it](https://github.com/DataDog/browser-sdk/pulls?q=is%3Apr+fix+coverage+in%3Atitle+), I am wondering if we could have some kind of check to prevent a regression here again